spec/views/profile/index.html.slim_spec.rb

```diff
@@ -20,7 +20,8 @@
       expect(rendered).not_to have_xpath("//input[@value='Disable']")
     end
 
-    it 'contains link to delete account' do
+    xit 'contains link to delete account' do
+      pending 'temporarily disabled until we figure out the MBUN to SSN mapping'
```

Comment:
you don't need both `pending` and `xit`.  In this case just `pending` should be fine (better actually i think).